cvat/apps/engine/migrations/0062_delete_previews.py

```diff
@@ -0,0 +1,60 @@
+import os
+import logging
+import sys
+import traceback
+
+from django.db import migrations
+from django.conf import settings
+
+def delete_previews(apps, schema_editor):
+    migration_name = os.path.splitext(os.path.basename(__file__))[0]
+    migration_log_file = '{}.log'.format(migration_name)
+    stdout = sys.stdout
+    stderr = sys.stderr
+    # redirect all stdout to the file
+    log_file_object = open(os.path.join(settings.MIGRATIONS_LOGS_ROOT, migration_log_file), 'w')
```

Comment:
I think, that the logging capabilities are necessary for all other migration files as well. I see it is the second while where we initialize logging.

0024_auto_20191023_1025.py